TodoList/TodoList.Api/Controllers/NodesController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Web.Http;
+using TodoList.Api.Models;
+
+namespace TodoList.Api.Controllers
+{
+    public class NodesController : ApiController
+    {
+        private NodeModel[] Nodes { get; }
+
+        public NodesController()
+        {
+            Nodes = InitializeData().ToArray();
+        }
+
+        [Route("api/v1/nodes")]
+        public async Task<IHttpActionResult> GetAsync()
+            => await Task.FromResult<IHttpActionResult>(Ok(Nodes));
+
+        [Route("api/v1/nodes/{id}")]
+        public async Task<IHttpActionResult> GetAsync(string id)
+            => await Task.FromResult<IHttpActionResult>(Ok(Nodes[0]));
+
+        [Route("api/v1/nodes/{text}")]
```

Comment:
Do not specify `text` in URL, read it from body of the HttpRequest (method signature should looke like this: `PostAsync([FromBody]string text)`)